src/lib.rs

```diff
@@ -52,8 +52,19 @@ pub struct Renderer {
     target: Option<TargetTexture>,
 }
 
+/// Parameters used in a single render that are configurable by the client.
+pub struct RenderParams {
+    /// The background color applied to the target. This value is only applicable to the full
+    /// pipeline.
+    pub clear_color: peniko::Color,
+
+    /// Dimensions of the rasterization target
+    pub width: u32,
+    pub height: u32,
+}
+
 impl Renderer {
-    /// Creates a new renderer for the specified device.
+    /// Creates a new renderer for the specified device with default options.
```

Comment:
`Renderer` is never created without 'default options' as far as I can see.

Reply:
Ah this comment is leftover from the first patch where I had the options at the Renderer level. Removed.